frontend/js/src/album/AlbumPage.tsx

```diff
@@ -334,8 +343,7 @@ export default function AlbumPage(): JSX.Element {
                   onClick={() => {
                     window.postMessage(
                       {
-                        brainzplayer_event: "play-listen",
-                        payload: listensFromAlbumsRecordingsFlattened,
+                        brainzplayer_event: "play-ambient-queue",
```

Comment:
Yes, currently this does clear up the queue, and starts playing the ambient queue.

According to me, we should have a clear queue and start playing the album immediately. Otherwise the play album button does not make any sense.

The user would click the button when they want they want to listen to the album immediately after discovering the album and tracks. This is how most of the music players out work.

Alternatively we can add another button near that button to add the album tracks to the queue. I did see that some of the players have a setting on what should play button do, play the track immediately or add it to the queue.

Reply:
OK, I agree with the playing immediately part.
I think then that we could have a new "add album to queue" button, but I guess we could also simply do it with the other button I suggested to add in the queue, to add the whole ambient queue to the regular queue.
It would essentially do the same thing when on an album page, but it's probably a good convenience to have both options